protocol/avdl/secret_ui.avdl

```diff
@@ -37,13 +37,15 @@ protocol secretUi {
 
   string getPaperKeyPassphrase(int sessionID, string username);
 
-  record SecretStorageFeature {
+  record Feature {
     boolean allow;
+    boolean respond; /* if we want the value back */
     string label;
   }
 
   record GUIEntryFeatures {
-    SecretStorageFeature secretStorage;
+    Feature storeSecret;
+    Feature showTyping;
```

Comment:
The gui is dumb in this case. If we don't like this architecture we can change it but right now this is just like an array of 'feature' which have a label and some control flags. We don't actually know (or care) what storeSecret means